src/Entity/User.php

```diff
@@ -124,6 +124,11 @@ public function addAssignment(Assignment $assignment): self
         return $this;
     }
 
+    public function hasLastAssignment(): bool
```

Comment:
I'm not really sure in method name.
Is `has...` method should really be named like `hasLastSomething`, since it's only checking if `something[]` is not empty?

Reply:
You are right, I forgot to update the method name. It will be `hasAssignments()`, good catch 👍